source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+
+How it works
+============
+The software updater now checks to see if there is any content in :file:`/usr/share/mix`.
+If content does exist, then it knows user content must be incorporated into the updater
+commands. Internally it does all the same operations to retrieve the upstream manifests
+and content, and will also do the same for the local content. Upstream Manifest.MoMs
+are still signature verified, so users are guaranteed all content not provided by their
+local system is secure and correct.
+
+When a user calls :file:`swupd-add-pkg`, the script performs all the necessary mixer
+commands and bookeeping to create a minimal mix. This mix contains only the new user
+bundle, and os-core (to keep track of the versioning and timestamp changes). To keep
+swupd from becoming too complex and multiplexing seperate Manifest.MoMs, the script also
+merges the upstream Manifest.MoM (once it is openssl verified), with the local Manifest.MoM.
+The result is a merged Manifest.MoM that contains any updates from upstream, with the
+local updates concantenated to it. The new merged manifest is then signed with the user's
+certificate, which swupd will use to verify the manifest is the one it should be.
+
+Consuming only one Manifest.MoM relieves some burden from the updater, which now can
+simply read it, and understand if a bundle originates from upstream, or from local. If
+a maniest is tagged as local in the Manifest.MoM, the updater will load that manifest
+from disc instead of going over the network, and all files from it will be marked as
+local as well. Thus when the file list is traversed for downloading, it "downloads"
+the local content directly from disc, while pulling the upstream changins from their
+regular source. All of this is done without any extra flags or configs needed, so users
+can expect swupd to operate the same as before, but now support multiple sources for
+content.
+
+While the version of the OS changes, it only does so by multiplying by 1000. This is
+so swupd can still easily determine the base OS version, for example 18220 becomes
+18220000, showing the base version ``18220`` and allowing for ``100`` new "mixes" or
+package adds within that version (increment by 10 x 100). By doing so, swupd can still
+check if there is a version > 18220, and if so, it will automatically generate the
+new Manifest.MoM for the user on the next update operation. This means autoupdate will
+still work, and users will continue recieving security updates per normal update
+schedule. On top of this, a mixed file will exist at :file:``/usr/share/defaults/swupd/mixed``
+signifying that the system is on an augmented version.
+
+How to use it
+=============
+Using the new feature is simple. Create a new folder :file:``/usr/share/mix/``, which
+is where the local update content will go. In that folder, create an "rpms" directory,
+and simply :command:``cp`` your RPMs into it.
```

Comment:
Need to note that you can't simply use any RPMs.  Has to be RPMs formatted for Clear.